zilencer/models.py

```diff
@@ -32,6 +32,7 @@ class RemotePushDeviceToken(AbstractPushDeviceToken):
     server: RemoteZulipServer = models.ForeignKey(RemoteZulipServer, on_delete=models.CASCADE)
     # The user id on the remote server for this device device this is
     user_id: int = models.BigIntegerField(db_index=True)
+    encrypted: bool = models.BooleanField(default=False)
```

Comment:
I don't like this name; it sounds like it's saying the device is encrypted (whatever that would mean).

One good name could be `encrypt_notifications` -- same as in the request the client makes when registering its token.

Another could be `require_encryption` -- reflecting the effect of the flag, which is to say "don't send plaintext messages here."